moulin/rouge/ext_utils.py

```
@@ -79,3 +79,14 @@ def mcopy(img: BinaryIO, file: str, name: str):
     args = ["mcopy", "-i", img.name, file, "::" + name]
 
     _run_cmd(args)
+
+
+def mmd(img: BinaryIO, name: str):
+    "Create a directory inside a vfat image"
+
+    # If specified folder already exists, then mmd asks
+    # user for possible action and returns error code.
+    # To avoid such behavior we us '-Ds' as 'skip' clash_option
+    # and ignore error code.
+    args = ["mmd", "-Ds",  "-i", img.name, "::" + name]
+    subprocess.run(args)
```

`_run_cmd`

---

Generally, it is not a good idea to ignore errors. Yes, we can expect that subsequent `mcopy` will fail, but nevertheless...

I want to propose third approach: build list of all directories/subdirectories beforehand. Scan all names in `self._files`, build list that includes all directories, remove duplicates, sort it and feed it to one `mmd` call.

---

Commit is reworked according to the proposal.

Now for this yaml:
```
files:
  "/zxc/qwe/asd/atf-uboot.ub": "atf-uboot.ub"
  "zxc/qwe/asd/boot.scr": "boot_xen.scr"
  "Image": "Image"
  "/as/system.dtb": "system.dtb"
  "/xc/u-boot.bin": "u-boot.bin"
  "uEnv.txt": "uEnv.txt"
```

We see in the output:
```
[INFO] Running mmd -i /tmp/tmp4jhaa2u9 ::/zxc ::/zxc/qwe ::/zxc/qwe/asd ::/as ::/xc
```